ci/ci.sh

```diff
@@ -83,6 +83,10 @@ else
 	echo VPP ping test failed
 	exit 1
 fi
+
```

Comment:
when the kind env is brought down, do these not get removed anyways?

Reply:
The kind env is destroyed at the end of the OVS test, here I am getting rid of VPP before OVS is started so that there is less hugepages needed overall